test/python/apps/exchange.py

```diff
@@ -186,12 +187,13 @@ def check_transaction(self) -> RAPDU:
 
     def _ticker_to_coin_payload(self, ticker) -> bytes:
         ticker_to_conf = {
+            "ETC": (ETC_CONF, ETC_CONF_DER_SIGNATURE, ETC_PACKED_DERIVATION_PATH),
             "ETH": (ETH_CONF, ETH_CONF_DER_SIGNATURE, ETH_PACKED_DERIVATION_PATH),
             "BTC": (BTC_CONF, BTC_CONF_DER_SIGNATURE, BTC_PACKED_DERIVATION_PATH),
             "LTC": (LTC_CONF, LTC_CONF_DER_SIGNATURE, LTC_PACKED_DERIVATION_PATH),
         }
-        assert ticker in ticker_to_conf
-        conf, signature, derivation_path = ticker_to_conf[ticker]
+        assert ticker.upper() in ticker_to_conf
+        conf, signature, derivation_path = ticker_to_conf[ticker.upper()]
```

Comment:
Why the `upper` ?
Eventually we may want to check the ticker against the values of  COIN_CONF[1:3] which contains the real ticker bytes sent to the app (full case here)

Reply:
LL APDUs keep using lower case versions of the tickers, and it seems to work, so I figured we don't care at that point.
But you're right I better check it thoroughly.

Reply:
Confirmed, that's okay: transaction are normalized [here](https://github.com/LedgerHQ/app-exchange/blob/develop/src/process_transaction.c#L65), so ticker could be lower or upper case.